src/components/input/input.ts

```diff
@@ -75,7 +74,7 @@ import { Platform } from '../../platform/platform';
 @Component({
   selector: 'ion-input',
   template:
-    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" (focus)="inputFocused($event)" [placeholder]="placeholder" class="text-input" [ngClass]="\'text-input-\' + _mode">' +
+    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" (focus)="inputFocused($event)" [placeholder]="placeholder" class="text-input" [ngClass]="\'text-input-\' + _mode" [attr.autocomplete]="autocomplete" [attr.autocorrect]="autocorrect" [attr.autocapitalize]="autocapitalize" [autofocus]="autofocus" [attr.list]="list" [attr.max]="max" [attr.maxlength]="maxlength" [attr.min]="min" [attr.minlength]="minlength" [readonly]="readonly" [required]="required" [spellcheck]="spellcheck" [attr.step]="step" [attr.tabindex]="tabindex" [attr.name]="name">' +
```

Comment:
Can you split the line into several ones like this one: https://github.com/manucorporat/ionic/blob/1a6c511ff7f900a7de9de82cee417973a1b32beb/src/components/searchbar/searchbar.ts#L38-L42

?